### PR TITLE
First PR

### PR DESCRIPTION
**Pull request by sagyMoveo**
## Description

This PR introduces improvements to the basic admin dashboard, including fetching collection names from the database and displaying data from selected collections. It also incorporates ESLint and adheres to Moveo's development guideline.

### Changes

- Fetch collection names from the database using a new API call.
- Enhance Dashboard component to allow selecting different collections.
- Table component is able to display data from any collection.
- Utilize useCallback to optimize rendering of keys and filtered keys.

### CODE REVIEW

File: src/shared/components/Table/Table.component.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  const apiFetch = useCallback(async () => {
+    const dataFromApi: ICollectionItemArray =
+      await ApiService.getAll(collectionName);
+    setData(dataFromApi);
+  }, [collectionName]);
+
+  useEffect(() => {
+    apiFetch();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [collectionName]);
```
Comment (nir-boost) [inline src/shared/components/Table/Table.component.tsx:29]:
```suggestion
  useEffect(() => {
    apiFetch();
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [collectionName, apiFetch]);
```

File: src/shared/components/Table/Table.component.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  }, []);
+
+  useEffect(() => {
+    if (data.length > 0) {
+      let keys = Object.keys(data[0]);
+      keys = filterKeys(keys, unwantedKeys);
+      setFilteredKeys(keys);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [data]);
```
Comment (nir-boost) [inline src/shared/components/Table/Table.component.tsx:46]: looks like useMemo

File: src/shared/parsers/CellParser.parser.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+// TODO: fix cell any type.
+export const renderCell = (cell, key: string) => {
+  if (key === 'image') {
```
Comment (nir-boost) [inline src/shared/parsers/CellParser.parser.tsx:3]: look like should extact const
```suggestion

  if (key === CELL_KEY.IMAGE) {
```

File: src/shared/parsers/CellParser.parser.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    return <img style={{ width: '140px' }} src={cell} alt='' />;
+  }
+  if (key === 'dishes') {
```
Comment (nir-boost) [inline src/shared/parsers/CellParser.parser.tsx:6]: same in all page

**Review by nir-boost — CHANGES_REQUESTED** [review]

File: src/shared/components/Table/Table.component.tsx (hunk 1 above)
Comment (sagyMoveo) [reply src/shared/components/Table/Table.component.tsx:29]: fixed

File: src/shared/parsers/CellParser.parser.tsx (hunk 3 above)
Comment (sagyMoveo) [reply src/shared/parsers/CellParser.parser.tsx:3]: fixed

File: src/shared/parsers/CellParser.parser.tsx (hunk 4 above)
Comment (sagyMoveo) [reply src/shared/parsers/CellParser.parser.tsx:6]: fixed

**Review by nir-boost — APPROVED** [review]

**Pull request merged**